README.md

```diff
@@ -11,47 +11,116 @@ ADaM in R Asset Library
-* It is expected for companies to adopt `{admiral}` that a company-specific extension package would likely be needed (e.g. `{admiral.roche}` or `{admiral.gsk}`)
-    * Consider this an opportunity to influence company-specific standards, as `{admiral}` offers a bridge towards a more industry-aligned implementation of ADaM
-* From the core `{admiral}` package covering common functions and mostly safety templates, there will be further package extensions dedicated to certain disease area endpoints (e.g. `{admiral.onco}` or `{admiral.hiv}`)
+To set expectations: It is not our target that `{admiral}` will ever provide all possible solutions
+for all ADaM datasets outside of study specific needs. It depends on the user's collaboration
+and contribution to help grow over time to a toolkit library that is robust, easy to use and
+has an across-industry focus. We do not see a coverage of 100% of all ADaM derivations as ever
+achievable - ADaM is endless.
+
+We will provide:
+* A toolbox of re-usable functions and utilities to create ADaM datasets using R scripts in a
+modular manner (an "opinionated" design strategy)
+* Pharmaceutical communities and companies are encouraged to contribute to `{admiral}` following
+the provided programming strategy and modular approach
+* Functions that are comprehensively documented and tested, including example calls - these are
+all listed in the [Reference section](https://pharmaverse.github.io/admiral/reference/index.html)
+* Vignettes on how to create ADSL, BDS and OCCUR datasets, including example scripts
+* Vignettes for ADaM dataset specific functionality (i.e. dictionary coding, date imputation,
+SMQs ...)
+
+## Types of Packages
+There will be 3 foreseeable types of `{admiral}` packages:
+* Core package - one package containing all core functions required to create ADaMs,
+usable by any company (i.e. general derivations, utility functions and checks for ADSL, OCCDS and BDS)
+* TA (Therapeutic Area) package extensions - one package per disease area with functions that are
+specific to algorithms and requirements for that particular TA (e.g. `{admiralonco}`)
+* Company package extensions - specific needs and plug-ins for the company, such as access to metadata
+(e.g. `{admiralroche}` or `{admiralgsk}`)
+
+## Admiral Manifesto
+For `{admiral}` and all extension packages, we prioritise providing our users with a **simple to adopt** toolkit
+that enables them to produce **readable** and **easily constructible** ADaM programs. The following explains
+our philosophy, which we try to adhere to across the `{admiral}` family of packages.
+There isn’t always a clear single, straight forward rule but there are guiding principles we adhere to.
+This manifesto helps show the considerations of our developers when making decisions.
+
+We have four priority design principles to achieve the main goal as follows:
+
```

Comment:
Could the four design principles be put at the top?  Maybe bulleted after main goal as follows: 

* Usability
* Simplicity
* Findability 
* Readability

Reply:
Robin had something similar in his draft, but i actually felt when it was down on the homepage that it reads better without repeating these so i took that out. Let's see what others think, but i was conscious of not putting too many new sections on the homepage for fear of scaring people off from actually reading it all

Reply:
@bms63 I would be fine as is right now. Is it ok to close this from your end?